deployment/step10.md

```diff
@@ -0,0 +1,14 @@
+# Deploy Application
+
+This step helps you to try out a sample application deployment.
+
+`kubectl apply -f $GOPATH/src/github.com/kubeedge/kubeedge/build/deployment.yaml`{{execute}}
+
+Then you can use below command to check if the application is normally running.
+
+`kubectl get pods`{{execute}}
+
```

Comment:
is it a good idea to say keep trying ? It may indicate that kubeedge has some issue and cant deploy applications in 1 go successfully

Reply:
This is because of environment delay in Katacoda.